_infra/helm/mock-eq/values.yaml

```diff
@@ -11,6 +11,16 @@ image:
   tag: latest
   pullPolicy: Always
 
+frontendConfig:
+  sslPolicy: mock-eq-ingress-ssl-policy
+
+ingress:
+  enabled: false
```

Comment:
Default if not overridden during a deployment is `false` which means no change to its current usage when deployed to the `dev` cluster.